server/render/index.js

```diff
@@ -120,6 +120,16 @@ export function render( element, key = JSON.stringify( element ), req ) {
 
 		return renderedLayout;
 	} catch ( ex ) {
+		// Log 1% of errors
+		if ( Math.random() < 0.01 ) {
+			logToLogstash( {
+				feature: 'calypso_ssr',
+				message: 'Exception thrown on render',
+				extra: {
+					error: ex,
+				},
+			} );
+		}
```

Comment:
Just log all the errors
```suggestion
		logToLogstash( {
			feature: 'calypso_ssr',
			message: 'Exception thrown on render',
			extra: {
				error: ex,
			},
		} );
```